src/main/java/com/wootecobook/turkey/comment/controller/api/CommentApiController.java

```diff
@@ -69,5 +70,19 @@ public ResponseEntity delete(@PathVariable Long id,
         final URI uri = linkTo(CommentApiController.class, postId).toUri();
         return ResponseEntity.noContent().location(uri).build();
     }
+
+    @GetMapping("/{id}/good")
+    public ResponseEntity<GoodResponse> good(@PathVariable Long id, UserSession userSession) {
```

Comment:
어노테이션 기반으로 동작할 수 있게 구현하면 어떤 이점이 있을까요?

```suggestion
    public ResponseEntity<GoodResponse> good(@PathVariable Long id, @LoginUser UserSession userSession) {
```

Reply:
명시적으로 로그인된 유저임을 확인할 수 있었습니다.
혹시 또 다른 이점이 있을까요?